server/lib/systemd/pbench-server.service.default

```diff
@@ -12,6 +12,7 @@ After=network.target httpd.service
 Type = simple
 PIDFile = /run/pbench-server/gunicorn.pid
 ExecStartPre = /bin/mkdir /run/pbench-server
+ExecStartPre = /bin/chown pbench.pbench /run/pbench-server
```

Comment:
I know the "." works with chown, but it might confuse recent Linux users, I would use ":", following the chown(1) manpage here: https://man7.org/linux/man-pages/man1/chown.1.html

Reply:
I didn't know that the "." separator was accepted, so thanks for the heads-up!